README.md

```diff
@@ -523,7 +523,11 @@ with `bax` filetype.
 
 #### Keybindings
 
-Plugins may be lazy-loaded on the use of keybindings/maps. Individual keybindings are specified either as a string (in which case they are treated as normal mode maps) or a table in the format `{mode, map}`.
+Plugins may be lazy-loaded on the use of keybindings/maps. Individual keybindings are specified either as a string (in which case they are treated as normal mode maps) or a table in one of the following formats:
```

Comment:
Here and in the `doc/` file, we need to change the wording a bit. It currently only talks about **individual** keybindings, but we want to say that you can specify *the whole set* of keybindings in one of these two table formats, not individual keybindings.

Reply:
Sorry for the late reply! Do you think they're clearer now?